docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:bionic
+
+RUN apt-get update && apt-get install -y git curl imagemagick librsvg2-bin gnupg && \
```

Comment:
Currently the clock CAPTCHA looks like this:
![download](https://user-images.githubusercontent.com/14208607/46110179-7831db80-c1a8-11e8-976b-18776ffd6f6c.png)
Which I suspect is due to missing dependencies for imagemagick, but I'm having trouble tracking it down. I've tried using `librsvg2-dev` instead, unfortunately still encountering it.

Reply:
I was playing around with the dependencies and extras for imagemagick, but nothing worked, until I tried building it from ubuntu:xenial, but I'm hesitant to go that far back. I'll try to check what's changed between xenial imagemagick and the current bionic